Tests/Core/ActivityIndicatorViewObserverTests.swift

```diff
@@ -0,0 +1,108 @@
+//
+//  ActivityIndicatorViewObserverTests.swift
+//  Operations
+//
+//  Created by Matthew Holden 16/04/2016.
+//
+//
+
+import Foundation
+import XCTest
+@testable import Operations
+
+enum AnimationState {
+    case Animating
+    case NotAnimating
+}
+
+class TestableActivityIndicatorView: ActivityIndicatorViewAnimationInterface {
+    typealias IndicatorAnimationStateDidChangeBlock =  (toState: AnimationState) -> Void
+
+    let animationStateDidChange: IndicatorAnimationStateDidChangeBlock
+
+    init(_ didChange: IndicatorAnimationStateDidChangeBlock) {
+        animationStateDidChange = didChange
+    }
+
+    func startAnimating() {
+        animationStateDidChange(toState: .Animating)
+    }
+
+    func stopAnimating() {
+        animationStateDidChange(toState: .NotAnimating)
+    }
+}
+
+
+class ActivityIndicatorViewObserverTests: OperationTests {
+
+    var operation: TestOperation!
+    var indicator: ActivityIndicatorViewAnimationInterface!
+    var animationStateChanges: [AnimationState]!
+
+    override func setUp() {
+        super.setUp()
+        animationStateChanges = [AnimationState]()
+        operation = TestOperation()
+        indicator = TestableActivityIndicatorView { [unowned self] animationState in
+            self.animationStateChanges.append(animationState)
+        }
```

Comment:
Maybe instead of collecting the state changes here, in the test case, it would make sense to just collect them in the `TestableActivityIndicatorView` instead.

Then your tests can just query for the animation state changes from the view directly. This will save you having to have this block here.